ocpi-toolkit-2.2.1/src/main/kotlin/com/izivia/ocpi/toolkit/modules/credentials/CredentialsClient.kt

```diff
@@ -12,38 +12,54 @@ import com.izivia.ocpi.toolkit.transport.domain.HttpRequest
  */
 class CredentialsClient(
     private val transportClient: TransportClient
-): CredentialsInterface {
+) : CredentialsInterface {
 
     override suspend fun get(tokenC: String): OcpiResponseBody<Credentials> =
         transportClient
             .send(
                 HttpRequest(method = HttpMethod.GET)
-                    .withDebugHeaders()
+                    .withRequiredHeaders(
+                        requestId = transportClient.generateRequestId(),
+                        correlationId = transportClient.generateCorrelationId()
+                    )
```

Comment:
this is not really part of the scope of this PR, but adding those parameters to the `withRequiredHeaders` makes it more obvious: it's tedious and error prone to have all the clients call this method to add the headers. I think we should move the responsibility to the transportClient to add the required headers in the send method. But I tihnk we can move forward with this PR and address this point in another PR, as well as the review of how authentication and tokens are used / named.

Reply:
issue created: https://github.com/IZIVIA/ocpi-toolkit/issues/18